sass/MagicBox/MagicBox.scss

```diff
@@ -120,6 +120,14 @@
     }
     .magic-box-suggestions.magic-box-hasSuggestion {
       display: block;
+      .coveo-magicbox-suggestions {
+        border: 1px solid $grey-500;
+      }
+      .coveo-suggestion-container {
```

Comment:
Ok so to remove the important I had to move the `border: none` here.
I tried a selector for the parent of `.coveo-magicbox-suggestions` to not have `.coveo-magicbox-suggestions` but I could get it to work so here we are.